lib/mpi/responses/profile_parser.rb

```diff
@@ -75,17 +81,17 @@ def parse
       # rubocop:disable Metrics/MethodLength
       # rubocop:disable Metrics/AbcSize
       def build_mvi_profile(patient)
```

Comment:
is it possible that the bulid_mvi_profile and build_relationship_mvi_profile could be refactored to take a "root" as an argument? Instead of having 2 different methods?

Reply:
Yeah that's not a bad point, there is a lot of repeated stuff here. Given that we don't fully know a lot about what to expect from the relationships array until we can actually grab it from MPI, I'm also considering severely scaling back the fields we parse from it, maybe even create an `MPIRelationshipProfile` model to differentiate

Reply:
Alright, lemme know how it looks now